modules/prebidServerBidAdapter/index.js

```diff
@@ -731,12 +731,19 @@ const OPEN_RTB_PROTOCOL = {
       }
 
       if (bidUserId.lipb && bidUserId.lipb.lipbid) {
-        request.user.ext.eids.push({
+        const liveIntent = {
```

Comment:
created a obj reference instead of adding object literal, because the segments property is added conditionally